cogs/Verification.py

```diff
@@ -166,6 +191,9 @@ async def enable_verification(self, ctx, suffix=None, role=None):
         :param role: the role to give users with that email verified (e.g. @students)
         :return:
         """
+        role = role
+        if not suffix:
+            await ctx.send("hi")
```

Comment:
Also just wondering the point in this considering the case of suffix not beign provided would be caught afterwards anyway